src/lib/features.js

```diff
@@ -882,7 +900,12 @@ var featureInfos =
             'STATUS',
             'WINDOW',
         ],
-        attributes: { 'char-increment-restriction': null, 'web-worker-restriction': null },
+        attributes:
+        {
+            'char-increment-restriction': null,
+            'web-worker-restriction':     null,
+            'non-ie-restriction':         null,
```

Comment:
Can we keep the attributes in alphabetical?

```suggestion
            'char-increment-restriction': null,
            'non-ie-restriction':         null,
            'web-worker-restriction':     null,
```